exercicios/exercicio-alunas/Denise/Script.js

```diff
@@ -0,0 +1,24 @@
+nome = prompt ('Vamos descobrir se você foi aprovado? Escreva seu nome.');
+nota1 = prompt ('Digite a nota que você obteve no primeiro trimestre.');
+nota1= parseInt (nota1);
```

Comment:
@DeniseL0pes aqui você pode usar Number() ao invés de parseInt()

Reply:
Olá @ptonella primeiro gostaria de agradecer pelo feedback. Fiquei muito feliz de você ter comentado cada pedaço com as melhorias a serem realizadas, isso é muito importante para mim. 

Fiquei apenas com duas dúvidas:
1 - Você mencionou que eu poderia usar Number(). Minha dúvida é, tanto faz um ou outro, ou neste caso o mais correto seria usar o Number?
2 - A mesma pergunta para o Math.round() ou Math.floor() ao invés de parseInt().

Desde já agradeço pelo retorno.

Reply:
1. No teu caso eu acho que ficaria mais simples se você trocar o parseInt() pelo Number(). A função Number vai garantir que o que o usuário digitou seja convertido para uma variável do tipo número. Lembrando que só estamos fazendo essa conversão porque estamos usando o prompt, quando os valores vierem do HTML você pode definir o tipo de dado diretamente no HTML.

2. O Math.round() é uma função que serve para arredondar números. Então se o valor da média for 6.8 o Math.round() vai arredondar esse valor para 7. 
Se você quiser saber mais sobre cada função, segue abaixo os links da documentação de cada uma:

[Math.round()](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Math/round)
[parseInt()](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/parseInt)

Reply:
Obrigada @ptonella :)